Cargo.toml

```
@@ -46,25 +46,31 @@ release_max_level_trace = []
 
 std = []
 
-# requires the latest stable
-# this will have a tighter MSRV before stabilization
-kv_unstable = ["value-bag"]
-kv_unstable_sval = ["kv_unstable", "value-bag/sval", "sval", "sval_ref"]
-kv_unstable_std = ["std", "kv_unstable", "value-bag/error"]
-kv_unstable_serde = ["kv_unstable_std", "value-bag/serde", "serde"]
+kv = []
+kv_sval = ["kv", "value-bag/sval", "sval", "sval_ref"]
+kv_std = ["std", "kv", "value-bag/error"]
+kv_serde = ["kv_std", "value-bag/serde", "serde"]
+
+# Deprecated: use `kv_*` instead
+# These `*_unstable` features will be removed in a future release
+kv_unstable = ["kv"]
+kv_unstable_sval = ["kv_sval"]
+kv_unstable_std = ["kv_std"]
+kv_unstable_serde = ["kv_serde"]
```

I think we need to enable these features.

```suggestion
kv_unstable = ["kv", "value-bag"]
kv_unstable_sval = ["kv_sval", "kv_unstable"]
kv_unstable_std = ["kv_std", "kv_unstable"]
kv_unstable_serde = ["kv_serde", "kv_unstable_std"]
```

This resolved most of the compilation errors for my crate, except for one. It can't find `log::kv::value::Visit`.